lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -483,6 +490,60 @@
             });
         };
 
+        StudentAdmin.prototype.override_problem_score_single = function() {
+            var defaultErrorMessage, fullDefaultErrorMessage, fullSuccessMessage,
+                problemToReset, score, sendData, successMessage, uniqStudentIdentifier,
+                that = this;
+            uniqStudentIdentifier = this.$field_student_select_grade.val();
+            problemToReset = this.$field_problem_select_single.val();
+            score = this.$field_select_score_single.val();
+            if (!uniqStudentIdentifier) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a student email address or username.')
```

Comment:
The text here is the error message that appears when the user hasn't entered one of these items. The user enters these values in the same place for all of the per-student admin tasks, so that interface hasn't changed.